pytest_xmlcomp/plugin.py

```diff
@@ -69,6 +87,9 @@ def __init__(self, xpath, parent, expresult, tree):
         print(self.root)
 
     def runtest(self):
+        """
+        Checks if the XPath from the JSON file is in the modified XML file.
+        """
```

Comment:
Two issues I think could be improved:

* Rephrase to: "Apply an XPaths to the modified XML file and check the result. "
* Amend the docstring with:

```
 :raises XPathError: result of XPath does not match with the modified XML
```